data/sql/updates/pending_db_world/rev_1592179496642830400.sql

```diff
@@ -0,0 +1,46 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1592179496642830400');
+
+CREATE TABLE IF NOT EXISTS `world_config` (
+  `type` TINYINT UNSIGNED NOT NULL DEFAULT 50,
+  `name` VARCHAR(255) NOT NULL,
+  `value` INT UNSIGNED NOT NULL,
```

Comment:
If the value is float?

Reply:
But there is no float in the config? Or maybe I am wrong?
When I did a quick look in the config I could only find values 0 - in the range of `INT UNSIGNED`

Reply:
Configuration has 4 types
1. string
https://github.com/azerothcore/azerothcore-wotlk/blob/34077115a5de7d640517caf8136f8648f8f195e3/src/server/game/World/World.h#L373
2. int
https://github.com/azerothcore/azerothcore-wotlk/blob/34077115a5de7d640517caf8136f8648f8f195e3/src/server/game/World/World.h#L200
3. bool 
https://github.com/azerothcore/azerothcore-wotlk/blob/34077115a5de7d640517caf8136f8648f8f195e3/src/server/game/World/World.h#L74
4. float
https://github.com/azerothcore/azerothcore-wotlk/blob/34077115a5de7d640517caf8136f8648f8f195e3/src/server/game/World/World.h#L182

Reply:
true, totally missed that

Reply:
I don't think that adding "type" column is the best way to do it, you can't store a "float" value inside an INT anyway.

I would go with value as a VARCHAR and then doing the conversion on core side based on type inside the core.

NOTE: If the core is not able to convert the DB value (maybe someone mispelled it) then we can throw an error inside the console and use the default value

Reply:
We also have textual configurations (MOTD for example) and we could have more of them in future

Reply:
Type was just a suggestion so we can sort the configs, it would have no use in the core.
see https://github.com/azerothcore/azerothcore-wotlk/pull/3130#discussion_r439885565

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/34077115a5de7d640517caf8136f8648f8f195e3/src/server/worldserver/worldserver.conf.dist#L6-L31
Just like this, but in the DB ^^